src/pilot/deployment-validation/sanity_test.sh

```diff
@@ -422,12 +447,21 @@ test_neutron_networking (){
 
   sleep 3
 
-  for floating_ip in ${floating_ips[@]}
-  do
-    # Test pinging the floating IP of the instance from the virtual router
-    # network namespace
-    ping_from_netns $floating_ip "qrouter-${router_id}"
-  done
+  if [ "DVR_ENABLED" != "False"]; then
+    for floating_ip in ${floating_ips[@]}
+    do
+      # Test pinging the floating IP of the instance from the virtual router
+      # network namespace
+      ping_from_snat_netns $floating_ip "qrouter-${router_id}"
```

Comment:
Shouldn't the above line be:
ping_from_snat_netns $floating_ip "snat-${router_id}"
?